bin/reversion.sh

```diff
@@ -43,4 +71,27 @@ fi
 
 echo "Changing from $VERSION to $NEWVERSION"
 
-find . -name 'pom.xml' -exec sed -i "s/<version>$VERSION<\/version>/<version>$NEWVERSION<\/version>/g" {} \;
+find . -name 'pom.xml' -exec sed -i "s/<version>$VERSION<\/version>/<version>$NEWVERSION<\/version>/g" {} \; -exec git add {} \;
```

Comment:
The documentations's ```pom.xml```  still declares ```<infinispan.version>9.2</infinispan.version>``` after replacement

Reply:
In ```server/integration/testsuite/pom.xml``` there is still ```<new.server.schema.version>9.2</new.server.schema.version>```

Reply:
In ```server/integration/versions/pom.xml``` there is  
```<version.org.infinispan.infinispan-management-console>9.2.0.Final</version.org.infinispan.infinispan-management-console>```

Which is fine, just a reminder that this should not be replaced :smile:

Reply:
The documentation property can be removed as it is ignored.

Reply:
I'd rather split this over multiple lines:

```
for f in $(find . ...); do
  sed -i ...
  git add ...
done
```